### ISSUE TITLE
Allow for compose transaction to clear an account 

### ISSUE THREAD

**Issue by jdetle** [trezor/connect#183]
```javascript
 let txData = await this.getBitcoinTxList(address)
    this.setProgress('Preparing transaction')
    let broadcastTransactions = this.state.broadcastTransactions
    
    let inputs = [
      {
        address_n: [44 | 0x80000000, 0 | 0x80000000, 2 | 0x80000000, 0, parseInt(index, 10)],
        prev_index: txData.count - 1,
        prev_hash: txData.last.hash,
      },
    ]
    let outputs = [
      {
        script_type: 'PAYTOADDRESS',
        amount: amount.minus(10000).toString(10),
        address: stripHexPrefix(this.state.home),
      },
    ]
    TrezorConnect.composeTransaction({ coin: 'Bitcoin', inputs, outputs, push: broadcastTransactions })
```

In outputs, I have 10000 satoshi subtracted to allocate some amount for fees, but I'd like to just have the ability to have fees draw from that amount. Seems like a general enough use case to make the request.

**Comment by szymonlesisz** [comment]
It doesn't work this way, because if user wants to send 1 btc it means that he wants to send 1 btc not 0.9 btc.
But i get the idea, i will add two more output types:
"send-max" and "op-return"

"send-max" will use all available inputs from account and calculate fee from them.

It should be done by the end of the week,
Thanks for your feedback

**Comment by jdetle** [comment]
Thanks so much! Appreciate it.

**Comment by szymonlesisz** [comment]
Available in next release (5.0.30)

Added two new output types, see [docs](https://github.com/trezor/connect/blob/develop/docs/methods/composeTransaction.md)
